lifecycle/support/infrastructure-monitoring-policy.md

```diff
@@ -1,5 +1,72 @@
+
+2. Infrastructure Monitoring Team:
+   - Collaborates with the Lead Support Engineer in monitoring the infrastructure components.
+   - Performs routine checks, analyzes performance metrics, and investigates anomalies or alerts generated by the monitoring tools.
+   - Follows incident response procedures and escalates issues to the appropriate teams or individuals for resolution.
+   - Assists in maintaining the documentation related to monitoring configurations and incident reports.
+
+3. Engineers:
+   - Utilize the monitoring tools to gain visibility into the performance and health of the infrastructure.
+   - Monitor the relevant metrics and alerts for their respective areas of responsibility.
+   - Collaborate with the Infrastructure Monitoring Team to address any identified issues or anomalies.
+
+## Monitoring Process
+
+1. Installation and Configuration:
+   - The Lead Support Engineer is responsible for the installation and configuration of monitoring systems and tools.
+   - Ensures that the monitoring systems are properly set up and integrated with the infrastructure components.
+
+2. Uptime Monitoring:
+   - Use StatusCake to monitor the uptime performance of websites.
+   - Configure notifications via Slack to promptly alert the team about downtime incidents.
+
+3. Incident Reporting:
+   - Utilize PagerDuty for reporting downtime incidents to the Support Team.
+   - Follow the defined incident reporting procedures for timely response and resolution.
+
+4. Security Monitoring:
+   - Employ Detectify, when paid for by clients, to perform periodic OWASP Top 10 scans and report security vulnerabilities.
+   - Collaborate with security personnel to address and resolve identified security vulnerabilities.
+
+5. Internal Server and Cost Monitoring:
+   - Leverage the bespoke internal Server monitoring and cost monitoring tool to track resource utilization, cost optimization, and unusual activity or outages.
+   - Configure alerts and notifications to promptly inform the Support Team about any anomalies or critical events.
+
+6. Training and Awareness:
+   - Utilize Udemy for Business to provide engineers with a wide range of security training materials.
+   - Encourage engineers to enhance their knowledge and skills in monitoring best practices and security measures.
+
+7. Access Management:
+   - Implement role-based access control mechanisms to manage access to monitoring tools at appropriate levels.
+   - Grant access privileges based on job responsibilities and requirements.
+
+## Non-Compliance and Consequences
+
+Non-compliance with this Infrastructure Monitoring Policy may result in data loss, compromised system security, or regulatory violations. Non-compliant individuals may be subject to disciplinary action.
+
+By following this Infrastructure Monitoring Policy, we ensure consistent and secure logging practices, protect log data integrity, and maintain compliance with data protection requirements.
```

Comment:
This section wasn't copied across correctly so I pasted it and tweaked it a little from the Logging policy